images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -1,5 +1,3 @@
-Import-Module (Join-Path $PSScriptRoot "SoftwareReport.Helpers.psm1") -DisableNameChecking
-
 function Get-OSName {
```

Comment:
nitpick: I think we should move this function to the Common.Helpers.psm1 or SoftwareReport.Helpers.psm1.